puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/foremanPluginsRelease.groovy

```diff
@@ -31,21 +31,17 @@ pipeline {
                         repoclosure('nightly', 'el7')
                         push_rpms('nightly', 'el7')
                     },
+                    '1.18/el7': {
+                        repoclosure('1.18', 'el7')
+                        push_rpms('1.18', 'el7')
+                    },
                     '1.17/el7': {
                         repoclosure('1.17', 'el7')
                         push_rpms('1.17', 'el7')
                     },
                     '1.16/el7': {
                         repoclosure('1.16', 'el7')
                         push_rpms('1.16', 'el7')
-                    },
-                    '1.15/el7': {
-                        repoclosure('1.15', 'el7')
-                        push_rpms('1.15', 'el7')
-                    },
-                    '1.15/f24': {
```

Comment:
In repoclosure there is a special case for f24 which can also be removed but doesn't hurt either.